_includes/social.html

```diff
@@ -0,0 +1,13 @@
+<ul class="social-media-list">
```

Comment:
Ahh, I don't know the best way to do it, but ideally, Minima would maintain the mapping of known services to their domains and would loop through that list looking for a corresponding key in the site's config.